website/src/views/components/KeyboardShortcuts.tsx

```diff
@@ -99,101 +99,83 @@ export class KeyboardShortcutsComponent extends React.PureComponent<Props, State
     });
 
     // Toggle night mode
-    this.bind('x', APPEARANCE, 'Toggle Night Mode', () => {
-      this.props.dispatch(toggleMode());
+    bind('x', APPEARANCE, 'Toggle Night Mode', () => {
+      dispatch(toggleMode());
 
       dispatch(
-        openNotification(`Night mode ${this.props.mode === DARK_MODE ? 'on' : 'off'}`, {
+        openNotification(`Night mode ${mode === DARK_MODE ? 'on' : 'off'}`, {
           overwritable: true,
         }),
       );
     });
 
     // Cycle through themes
-    this.bind('z', APPEARANCE, 'Previous Theme', () => {
+    function notifyThemeChange() {
+      const theme = themes.find((t) => t.id === themeId);
+      if (theme) {
+        dispatch(
+          openNotification(`Theme switched to ${theme.name}`, {
+            timeout: THEME_NOTIFICATION_TIMEOUT,
+            overwritable: true,
+          }),
+        );
+      }
+    }
+
+    bind('z', APPEARANCE, 'Previous Theme', () => {
       dispatch(cycleTheme(-1));
-      this.notifyThemeChange();
+      notifyThemeChange();
     });
 
-    this.bind('c', APPEARANCE, 'Next Theme', () => {
+    bind(['c', '0'], APPEARANCE, 'Next Theme', () => {
```

Comment:
Hmm... I know I wrote this but I don't know why I did it. Can anyone make sense of this? LOL. Otherwise let's undo this

```suggestion
    bind('c', APPEARANCE, 'Next Theme', () => {
```

Reply:
Yeah, seems like I was messing around to test some hypothesis (i.e. was drunk)